core/src/main/java/io/blockv/core/model/DiscoverPack.kt

```diff
@@ -10,14 +10,14 @@
  */
 package io.blockv.core.model
 
-class DiscoverGroup(
+class DiscoverPack(
```

Comment:
Out of interest, why it the Pack returned from inventory and the DiscoverPack from discover different?

Reply:
Discover query returns the additional count field, so the discover pack maps that value where pack doesn't